server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/CounterResource.java

```diff
@@ -78,16 +56,20 @@
     private static final OperationDefinition COUNTER_DECREASE = buildOperation("counter-decrease").build();
 
     private final boolean runtimeRegistration;
-    private final ResolvePathHandler resolvePathHandler;
 
     public CounterResource(PathElement pathElement, ResourceDescriptionResolver descriptionResolver,
```

Comment:
No need for this constructor anymore, just do the variable assignment and calls to super in the constructor:
 ```java
public CounterResource(PathElement pathElement, ResolvePathHandler resolvePathHandler,
            boolean runtimeRegistration) {
...
}
```